src/docs/manual/03_task_publishToConfluence.adoc

```diff
@@ -88,17 +88,45 @@ If you need to provide a proxy to access Confluence, you may set a map with keys
 
 useOpenapiMacro::
 If this option is present and equal to "confluence-open-api" then any source block marked with class openapi will be wrapped in Elitesoft Swagger Editor macro:  (see https://marketplace.atlassian.com/apps/1218914/open-api-swagger-editor-for-confluence[Elitesoft Swagger Editor]) +
-For backward compatibility: If this option is present and equal to `true`, then again the Elitesoft Swagger Editor macro will be used. +
-If this option is present and equal to "open-api" then any source block marked with class openapi will be wrapped in Open API Documentation for Confluence macro:  (see https://marketplace.atlassian.com/apps/1215176/open-api-documentation-for-confluence[Open API Documentation for Confluence]). A download source (yaml) button is shown by default. +
-This is how you'd include your openapi YAML file: +
-+
+For backward compatibility: If this option is present and equal to `true`, then again the Elitesoft Swagger +
```

Comment:
I get the feeling that there is something wrong with the documentation at this point. Can you please double-check?

Reply:
I changed the documentation for this in order to explain how to use the configuration for the OpenAPI Documentation Plugin as this differs from Confluent OpenAPI Plugin.

![Bildschirmfoto 2021-09-24 um 14 08 03](https://user-images.githubusercontent.com/88428763/134671864-56af10d7-197d-41f4-928b-b7c2efd3ea4a.png)

I can revoke the changes, but I thought it's worth to explain how the plugin can be used.

Reply:
ok